### ISSUE TITLE
meta-rauc-nxp imx8x: Nothing PROVIDES 'u-boot-fslc'

### ISSUE THREAD

**Issue by roby2014** [rauc/meta-rauc-community#90]
hello. im trying to add rauc ota support for my imx8x board. following this: https://github.com/rauc/meta-rauc-community/tree/master/meta-rauc-nxp
```
INIT_MANAGER = "systemd"
IMAGE_INSTALL:append = " rauc"
DISTRO_FEATURES:append = " rauc"
IMAGE_FSTYPES:append = " ext4"
IMAGE_BOOT_FILES:append = " boot.scr"
PREFERRED_PROVIDER_virtual/bootloader="u-boot-fslc"
PREFERRED_PROVIDER_u-boot="u-boot-fslc"
```
but im getting
```
ERROR: Nothing PROVIDES 'u-boot-fslc' (but /home/roby/builder/build/../layers/meta-rauc/recipes-core/rauc/rauc_1.11.1.bb DEPENDS on or otherwise requires it)
```
im a bit confused since here: https://git.yoctoproject.org/meta-freescale/tree/conf/machine/include/imx8x-mek.inc?h=kirkstone
it says:
```
# This machine is not supported by u-boot-fslc, so we force it to use
# u-boot-imx here.
IMX_DEFAULT_BOOTLOADER = "u-boot-imx"
UBOOT_SUFFIX = "bin"
```
Any idea?

**Comment by leon-anavi** [comment]
Hi @roby2014,

As of the moment layer meta-rauc-nxp supports only HummingBoard with i.MX6 thanks to a contribution from @AtanasBunchev. Modifications of the layer will be required to support additional boards.

The particular error you are facing for your target machine is caused by this dependency:
https://github.com/rauc/meta-rauc-community/blob/c13717ce953fea8329aac0922cdceeb69bf02ee6/meta-rauc-nxp/recipes-core/rauc/rauc_%25.bbappend#L5

Best regards, Leon

**Comment by roby2014** [comment]
Thanks for your quick answer.
Shouldn't there be maybe a list with supported/tested boards?

**Comment by leon-anavi** [comment]
> Thanks for your quick answer. Shouldn't there be maybe a list with supported/tested boards?

Yes, you are right. The README needs more details about the supported and tested boards.

A step by step tutorial for HummingBoard (aka Yocto/OE machine cubox-i) is available here:
https://www.konsulko.com/settingup-rauc-hummingboard-ota-software-update

**Comment by ejoerns** [comment]
The [README](https://github.com/rauc/meta-rauc-community/blob/master/meta-rauc-nxp/README.rst) lists instructions for supported boards and the `DEPENDS` on `u-boot-fslc` was removed in ff3286bd904e666d2f7352954c5fb243ee30f9e1. So maybe we can close this?

**Comment by ejoerns** [comment]
https://github.com/rauc/meta-rauc-community/pull/142 could also be a slight improvement in the visibility of supported boards.
